ethicalads-theme/templates/ea/niche-targeting.html

```diff
@@ -0,0 +1,146 @@
+
+        <!-- Icon -->
+        <div class="icon text-primary mb-3">
+          <svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M0 0h24v24H0z"></path><path d="M19 11h2a1 1 0 010 2h-2a1 1 0 010-2zM3 11h2a1 1 0 010 2H3a1 1 0 010-2zm9-9a1 1 0 011 1v2a1 1 0 01-2 0V3a1 1 0 011-1zm0 16a1 1 0 011 1v2a1 1 0 01-2 0v-2a1 1 0 011-1z" fill="#335EEA" opacity=".3"></path><circle fill="#335EEA" cx="12" cy="12" r="3"></circle></g></svg>
+        </div>
+
+        <!-- Heading -->
+        <h3>Bespoke contextual targeting</h3>
+
+        <!-- Text -->
+        <p class="text-gray-800 mb-6 mb-md-0">Target your next campaign to your exact developer niche with contextual targeting created just for you based on your landing and product pages.</p>
+
+      </div>
+      <div class="col-12 col-md-4 mb-5 mb-md-0 text-center" data-aos="fade-up" data-aos-delay="50">
+
+        <!-- Icon -->
+        <div class="icon text-primary mb-3">
+          <svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M0 0h24v24H0z"></path><path d="M5 10.5C5 6 8 3 12.5 3S20 6.75 20 10.5c0 2.333-2.176 5.536-6.529 9.61a1.5 1.5 0 01-1.994.05C7.16 16.507 5 13.287 5 10.5zm7.5 1.5a2.5 2.5 0 100-5 2.5 2.5 0 000 5z" fill="#335EEA"></path></g></svg>
+        </div>
+
+        <!-- Heading -->
+        <h3>Laser-focused</h3>
+
+        <!-- Text -->
+        <p class="text-gray-800 mb-6 mb-md-0">Using a machine learning model to find similar pages, we only show your ad next to the most relevant content.</p>
+
+      </div>
+      <div class="col-12 col-md-4 mb-5 mb-md-0 text-center" data-aos="fade-up" data-aos-delay="100">
+
+        <!-- Icon -->
+        <div class="icon text-primary mb-3">
+          <svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M0 0h24v24H0z"></path><path d="M6 9v6a3 3 0 003 3h6v.818C15 20.232 14.232 21 12.818 21H5.182C3.768 21 3 20.232 3 18.818v-7.636C3 9.768 3.768 9 5.182 9H6zm11 7v-6a3 3 0 00-3-3H8v-.818C8 4.768 8.768 4 10.182 4h7.636C19.232 4 20 4.768 20 6.182v7.636C20 15.232 19.232 16 17.818 16H17z" fill="#335EEA" opacity=".3"></path><path d="M9.273 9h4.454C14.552 9 15 9.448 15 10.273v4.454c0 .825-.448 1.273-1.273 1.273H9.273C8.448 16 8 15.552 8 14.727v-4.454C8 9.448 8.448 9 9.273 9z" fill="#335EEA"></path></g></svg>
+        </div>
+
+        <!-- Heading -->
+        <h3>Transparent</h3>
+
+        <!-- Text -->
+        <p class="text-gray-800 mb-0">Our ad reporting is completely transparent and shows you exactly where your ads were shown in terms of geography, publishers, and content each and every day.</p>
+
+      </div>
+    </div> <!-- / .row -->
+
+    <p class="mt-6 text-center">
+      <a href="/advertisers/similar-pages/" class="btn btn-outline-primary">Try our custom niche targeting</a>
```

Comment:
Should this pre-fill a URL? 

```suggestion
      <a href="/advertisers/similar-pages/" class="btn btn-outline-primary">Try our custom niche targeting demo</a>
```